package.json

```diff
@@ -9,13 +9,35 @@
   "categories": [
     "Other"
   ],
-  "activationEvents": [],
+  "activationEvents": [
+    "onLanguage:r",
+    "onLanguage:rd",
+    "onLanguage:rmd",
+    "onLanguage:debian-control.r",
+    "onLanguage:namespace.r",
+    "onLanguage:buildignore.r",
+    "workspaceContains:**/*.{rproj,Rproj,r,R,rd,Rd,rmd,Rmd}"
+  ],
   "main": "./out/extension.js",
   "contributes": {
     "commands": [
       {
-        "command": "vscode-flowr.helloWorld",
-        "title": "Hello World"
+        "command": "vscode-flowr.slice.cursor",
```

Comment:
we should definitely find better interface points for flowR these were just my quick-and-dirty hacks. but - for now - they are fine.